Sming/Makefile-rboot.mk

```diff
@@ -12,6 +12,9 @@
 RBOOT_BIG_FLASH  ?= 1
 RBOOT_TWO_ROMS   ?= 0
 RBOOT_RTC_ENABLED ?= 0
+RBOOT_GPIO_ENABLED ?= 0
+RBOOT_IROM_CHKSUM ?= 0
```

Comment:
You need to add `-iromchksum` to the parameters passed to esptool2 when the application rom is created if RBOOT_IROM_CHKSUM is enabled (set to 1). Probably want something like this in the makefile:
```
ifeq ($(RBOOT_IROM_CHKSUM), 1)
  RBOOT_E2_USER_ARGS += -iromchksum
endif
```